templates/zerver/help/include/sidebar_index.md

```diff
@@ -52,6 +52,7 @@
 * [Configure default view](/help/configure-default-view)
 * [Enable full width display](/help/enable-full-width-display)
 * [Display the buddy list on narrow screens](/help/move-the-users-list-to-the-left-sidebar)
+* [Organization logo as favicon](/help/organization-logo-as-favicon)
```

Comment:
Maybe match this to the header of the `/help` article so that it starts with a verb: 'Use organization logo as favicon'. I checked with the inspector and I think it will still fit in the sidebar as one line (see image).

<img width="270" alt="Screen Shot 2021-11-02 at 4 12 46 PM" src="https://user-images.githubusercontent.com/63245456/139875297-b6982351-6382-450f-8f06-26884984292c.png">